promql/value.go

```diff
@@ -92,6 +92,8 @@ func (s Series) String() string {
 type FPoint struct {
 	T int64
 	F float64
+	// OrigT is the original sample timestamp.
+	OrigT int64
```

Comment:
Do we need to add this here, if its only really used in samples that are read inside the info function? It feels like an implementation detail of the info function that we should expose only locally in it and not in an exported struct.

Reply:
Note: we can reconsider in followup cleanup RP

Reply:
Removing these fields.